tests/download_emulators.sh

```diff
@@ -3,6 +3,10 @@ SITE="https://firmware.corp.sldev.cz/upgrade_tests/"
 cd "$(dirname "$0")"
 
 # download all emulators without index files, without directories and only if not present
-wget --no-verbose --no-clobber --no-parent --cut-dirs=2 --no-host-directories --recursive --reject "index.html*" -P emulators/ $SITE
+wget -e robots=off --no-verbose --no-clobber --no-parent --cut-dirs=2 --no-host-directories --recursive --reject "index.html*" -P emulators/ $SITE
 
 chmod u+x emulators/trezor-emu-*
+
+if [[ `uname -a` =~ "NixOS" ]]; then
+  cd emulators && nix-shell --run "autoPatchelf trezor-emu* > /dev/null 2>&1"
```

Comment:
Why not just using plain
```
nix-shell --run "autoPatchelf trezor-emu*"
```
?

Is there any benefit to redirecting the outputs?

Reply:
I've copied it from adminkos to be honest. It is true that the output is quite verbose, but I guess they hid it for "ordinary" users and we actually want the output. Fixed.